### PR TITLE
ZCS-11296:Added Java 17 for zmdiaglog

### PR DESCRIPTION
**Pull request by umagmrit**

### CODE REVIEW

File: src/libexec/zmdiaglog [hunk 1]
```diff
@@ -91,6 +91,9 @@ sub get_java_version() {
     open( JAVA, "$JAVA -version 2>&1 |" )
       || die "Cannot determine java version: $!";
     my $version_line = <JAVA>;
+    if ( $version_line =~ /"17\.0/ ) {
+	$version = "17.0";
```
Comment (dasiyogesh) [inline src/libexec/zmdiaglog:95]: correct indentation

**Review by zimsuchitgupta — APPROVED** [review]

File: src/libexec/zmdiaglog [hunk 2]
```diff
@@ -91,6 +91,9 @@ sub get_java_version() {
...
+    }
     if ( $version_line =~ /"13\.0/ ) {
```
Comment (silentsakky) [inline src/libexec/zmdiaglog:97]: should this be converted to elsif?

File: src/libexec/zmdiaglog (hunk 1 above)
Comment (umagmrit) [reply src/libexec/zmdiaglog]: done

File: src/libexec/zmdiaglog (hunk 2 above)
Comment (umagmrit) [reply src/libexec/zmdiaglog]: yes. Updated code.

**Review by dasiyogesh — APPROVED** [review]

**Pull request merged**